core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/frameCol.kt

```diff
@@ -50,26 +50,26 @@ public interface FrameColColumnsSelectionDsl {
      * }
      *
      * {@set [DslGrammarTemplate.PlainDslFunctionsArg]
-     *  {@include [PlainDslName]}`[`**`<`**{@include [DslGrammarTemplate.ColumnTypeRef]}**`>`**`]`**`(`**{@include [DslGrammarTemplate.ColumnRef]}` | `{@include [DslGrammarTemplate.IndexRef]}**`)`**
+     *  {@include [PlainDslName]}`[`**`<`**{@include [DslGrammarTemplate.ColumnTypeRef]}**`>`**`]`**`(`**{@include [DslGrammarTemplate.ColumnRef]}`  |  `{@include [DslGrammarTemplate.IndexRef]}**`)`**
      * }
      *
      * {@set [DslGrammarTemplate.ColumnSetFunctionsArg]
      *  {@include [Indent]}{@include [ColumnSetName]}**`(`**{@include [DslGrammarTemplate.IndexRef]}**`)`**
      * }
      *
      * {@set [DslGrammarTemplate.ColumnGroupFunctionsArg]
-     *  {@include [Indent]}{@include [ColumnGroupName]}`[`**`<`**{@include [DslGrammarTemplate.ColumnTypeRef]}**`>`**`]`**`(`**{@include [DslGrammarTemplate.ColumnRef]}` | `{@include [DslGrammarTemplate.IndexRef]}**`)`**
+     *  {@include [Indent]}{@include [ColumnGroupName]}`[`**`<`**{@include [DslGrammarTemplate.ColumnTypeRef]}**`>`**`]`**`(`**{@include [DslGrammarTemplate.ColumnRef]}`  |  `{@include [DslGrammarTemplate.IndexRef]}**`)`**
      * }
      */
     public interface Grammar {
 
-        /** [**frameCol**][ColumnsSelectionDsl.frameCol] */
+        /** [**`frameCol`**][ColumnsSelectionDsl.frameCol] */
         public interface PlainDslName
 
-        /** .[**frameCol**][ColumnsSelectionDsl.frameCol] */
+        /** __`.`__[**`frameCol`**][ColumnsSelectionDsl.frameCol] */
```

Comment:
How this works? What kind of formatting?

Reply:
`__` is the same as `**` :) so in this case it makes the period bold+code style.
I switched to using `__` instead of `**` here because:
- kdocs don't like `*` at the start. It doesn't always recognize the difference between `*` indicating the KDoc or `*` for making stuff bold
- If this piece of text is copied over next to something else bold, like ``**`singleColumn`**{@include [ColumnGroupName]}``, the result would become ``**`singleColumn`****`.`**[**`frameCol`**]...`` and break markdown because of the `****`. 
Like this:
**`singleColumn`****`.`**[**`frameCol`**]()...
With dashes, instead it becomes  ``**`singleColumn`**__`.`__[**`frameCol`**]...`` which renders correctly: 
**`singleColumn`**__`.`__[**`frameCol`**]()...